modules/checklist/module.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+namespace Elementor\Modules\Checklist;
+
+use Elementor\Core\Base\Module as BaseModule;
+use Elementor\Core\Experiments\Manager;
+use Elementor\Core\Isolation\Wordpress_Adapter;
+use Elementor\Plugin;
+use Elementor\Utils;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Module extends BaseModule {
+	const EXPERIMENT_ID = 'launchpad-checklist';
+	const DB_OPTION_KEY = 'e-checklist';
```

Comment:
should it be `elementor_checklist`? 
As I see other names are in format `elementor_*`

Reply:
Sure!